sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -100,27 +117,43 @@ export interface AzureMachineLearningSkill extends BaseSearchIndexerSkill {
 }
 
 // @public
-export interface AzureOpenAIEmbeddingSkill extends BaseSearchIndexerSkill {
-    apiKey?: string;
-    authIdentity?: SearchIndexerDataIdentity;
-    deploymentId?: string;
+export interface AzureMachineLearningVectorizer extends BaseVectorSearchVectorizer {
+    aMLParameters?: AzureMachineLearningVectorizerParameters;
```

Comment:
the casing here feels off - AMLParameters? 

or even just Parameters / VectorizerParameters since they are parameters to the AzureMachineLearningVectorizer?